glyphs2fontir/src/source.rs

```diff
@@ -258,20 +276,97 @@ impl Work for StaticMetadataWork {
     }
 }
 
+fn design_location(axes: &[ir::Axis], master: &FontMaster) -> DesignLocation {
+    axes.iter()
+        .zip(master.axes_values.as_ref().unwrap())
+        .map(|(axis, pos)| (axis.tag.clone(), DesignCoord::new(pos.into_inner() as f32)))
+        .collect()
+}
+
+fn to_normalized(axes: &[ir::Axis], design_location: &DesignLocation) -> NormalizedLocation {
+    let axes: HashMap<_, _> = axes.iter().map(|a| (&a.tag, a)).collect();
+    design_location
+        .iter()
+        .map(|(tag, dc)| {
+            (
+                tag.clone(),
+                dc.to_normalized(&axes.get(tag).unwrap().converter),
+            )
+        })
+        .collect()
+}
+
 struct GlyphIrWork {
     glyph_name: String,
+    font: Arc<Font>,
+    master_indices: Arc<HashMap<String, usize>>,
```

Comment:
they are not 'indices' as in 0, 1, 2.. They are (UUID) identifiers, so I would call this `master_ids` for clarity's sake